ui/src/app/kvm/views/KVMList/KVMListTable/CPUColumn/CPUColumn.tsx

```diff
@@ -3,9 +3,14 @@ import { useSelector } from "react-redux";
 
 import { pod as podSelectors } from "app/base/selectors";
 import Meter from "app/base/components/Meter";
+import RootState from "app/base/reducers/types";
 
-const CPUColumn = ({ id }) => {
-  const pod = useSelector((state) => podSelectors.getById(state, id));
+type Props = { id: number };
+
+const CPUColumn = ({ id }: Props): JSX.Element => {
+  const pod = useSelector((state: RootState) =>
```

Comment:
All of these selectors hooks were causing TS issues, namely `Expected 1 arguments, but got 2.` To fix it I've converted the selector files to valid TS, but maybe there's a more progressive way to do it.